<ui version="4.0" >
 <class>LocalPlayerWidget</class>
 <widget class="QWidget" name="LocalPlayerWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>201</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QFrame" name="mainFrame" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet" >
      <string notr="true" >QFrame#mainFrame
{
	background-color: rgba(0,0,0,32);
	border: 5px solid rgba(0,0,0,0);
}

</string>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth" >
      <number>2</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4" >
      <item>
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <widget class="client::PlayerCharacterWidget" native="1" name="mp_characterWidget" />
          </item>
          <item>
           <spacer name="verticalSpacer_3" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="1" >
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" >
            <item>
             <spacer name="horizontalSpacer_2" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="client::CardWidget" native="1" name="roleWidget" />
            </item>
            <item>
             <spacer name="horizontalSpacer_3" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <spacer name="horizontalSpacer_4" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="styleSheet" >
               <string notr="true" >background-color: rgba(0, 0, 0, 64);
color: white;
font: oblique 7pt "DejaVu Sans";</string>
              </property>
              <property name="text" >
               <string>TODO</string>
              </property>
              <property name="textFormat" >
               <enum>Qt::LogText</enum>
              </property>
              <property name="alignment" >
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="2" >
         <layout class="QVBoxLayout" name="verticalLayout_4" >
          <property name="spacing" >
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="mp_buttonEndTurn" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="text" >
             <string>T</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mp_buttonPass" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="text" >
             <string>P</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mp_buttonDiscard" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="text" >
             <string>D</string>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="3" >
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <item>
           <widget class="QLabel" name="mp_labelPlayerName" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font" >
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="styleSheet" >
             <string notr="true" >color: white;</string>
            </property>
            <property name="text" >
             <string>{PLAYERNAME}</string>
            </property>
            <property name="scaledContents" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" >
        <item>
         <widget class="client::CardList" native="1" name="mp_table" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="client::CardList" native="1" name="mp_hand" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>client::CardWidget</class>
   <extends>QWidget</extends>
   <header>cardwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>client::PlayerCharacterWidget</class>
   <extends>QWidget</extends>
   <header>playercharacterwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>client::CardList</class>
   <extends>QWidget</extends>
   <header>cardlist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
